src/pyEQL/solution.py

```diff
@@ -3228,3 +3230,34 @@ def get_salt_list(self):  # pragma: no cover
         :py:meth:`get_salt_dict`
         """
         return self.get_salt_dict()
+
+    @classmethod
+    def from_preset(cls, preset: str) -> Solution:
+        """Instantiate a solution from a preset composition
+
+        Args:
+            preset (str): a named preset e.g. 'seawater' or the path to a .yaml file that defines the desired preset
+
+        Returns:
+            Solution
+        """
+        # Path to the YAML file corresponding to the preset
+        yaml_path = os.path.join("presets", f"{preset}.yaml")
+
+        # Check if the file exists
+        if not os.path.exists(yaml_path):
+            raise FileNotFoundError(f"File '{yaml_path}' not found!")
+
+        # Load the yaml file corresponding to the preset
+        with open(yaml_path) as file:
+            data = yaml.load(file, Loader=yaml.FullLoader)
+
+        # Extract data from the yaml file
+        temperature = data["temperature"]
+        pressure = data["pressure"]
+        pH = data["pH"]
+        solutes = data["solutes"]
+        # solutes = [[solute["name"], solute["concentration"]] for solute in solutes]
+
+        # Create and return a Solution object
+        return cls(solutes=solutes, temperature=temperature, pressure=pressure, pH=pH)
```

Comment:
I believe you can streamline this code by using dictionary unpacking. If the yaml file is read in as a `dict` (which I believe it is), then you should just be able to use

```
return cls(**data)
```

Reply:
Actually, an even better solution here would be to follow the pattern illustrated in the [documentation for saving to json files](https://pyeql.readthedocs.io/en/latest/serialization.html#loading-from-a-json-file)

`monty.serialization.loadfn` can load both json and yaml files (and you can use `dumpfn` to save to either .json or .yaml as well - this is how I recommend creating the .yaml files for presets).

So if you use `loadfn`, all you have to do is
```
return loadfn(<path to preset>)
```

At some poin, I would also like to add `Solution.to_file()` and `Solution.from_file()` methods to make the `dumpfn` / `loadfn` process even easier. Feel free to give that a try as well (but don't feel obligated as that's a significant expansion in scope for this PR).